### ISSUE TITLE
Working with babel 7

### ISSUE THREAD

**Issue by asturur** [thgreasi/babel-plugin-system-import-transformer#16]
Since i upgraded from babel 6 to babel 7 i cannot make this transform work anymore.

I m running tests with jest, and while webpack 3 Is able to remove the System.transforms for me, when babel-jest run this plugin i get errors.

```
  ● Test suite failed to run

    Unknown node type: "undefined"

      at cloneNode (node_modules/@babel/types/lib/clone/cloneNode.js:44:11)
      at Object.clone (node_modules/@babel/types/lib/clone/clone.js:13:33)
      at getImportedModuleFile (node_modules/babel-plugin-system-import-transformer/dist/moduleImportHelper.js:34:57)
      at getImportModuleName (node_modules/babel-plugin-system-import-transformer/dist/moduleImportHelper.js:46:28)
      at new SystemImportExpressionTransformer (node_modules/babel-plugin-system-import-transformer/dist/SystemImportExpressionTransformer.js:33:68)
      at PluginPass.CallExpression (node_modules/babel-plugin-system-import-transformer/dist/index.js:25:31)
      at newFn (node_modules/@babel/traverse/lib/visitors.js:193:21)
      at NodePath._call (node_modules/@babel/traverse/lib/path/context.js:53:20)
      at NodePath.call (node_modules/@babel/traverse/lib/path/context.js:40:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:88:12)
```

Would look like that something in @babel/types is not good anymore to clone the nodes as passed from this plugin.

Any clue?

**Comment by thgreasi** [comment]
That's expected since I didn't have the time to upgrade this repo to work with babel 7 yet.
Thanks for bringing this up.

**Comment by kopax** [comment]
We use the wonderful documentationjs and all our webpack projects are now broken since babel 7. Sorry to ask one more time, is there any status update?

**Comment by thgreasi** [comment]
No unfortunately I didn't have the time to start working on this.
I will try to do so soon, but PRs are always welcome.

**Comment by kopax** [comment]
Ok. hope you will have time soon. I am not a babel expert, I've been only using it and I am not trained in any of the transpilation steps. I wish it would be my grade so I could help. If you think a few indications could still help me contribute for this one just let me know how I will gladly try.

**Comment by thgreasi** [comment]
@asturur, @kopax just released v4.0.0 that adds support for babel v7.
If you do like this package, consider starring the repo, otherwise I can't easily reason about its adoption and prioritize such upgrades.
